google/cloud/storage/benchmarks/throughput_experiment_test.cc

```diff
@@ -64,11 +64,9 @@ TEST_P(ThroughputExperimentIntegrationTest, Upload) {
                                       /*enable_crc32c=*/false,
                                       /*enable_md5=*/false};
     auto result = e->Run(bucket_name_, object_name, config);
-    EXPECT_STATUS_OK(result.status);
-    if (result.status.ok()) {
-      auto status = client->DeleteObject(bucket_name_, object_name);
-      EXPECT_STATUS_OK(status);
-    }
+    ASSERT_STATUS_OK(result.status);
+    // This is not checked as it is just for cleanup.
+    (void)client->DeleteObject(bucket_name_, object_name);
```

Comment:
Q.  Would it be "nicer" to go back to the original formulation, but allow either OK or NOT_FOUND from the `DeleteObject()`?

Reply:
I am happy either way, and we can always change it. Done.